yarn/Dockerfile

```diff
@@ -1,16 +1,16 @@
 # Use the base App Engine Docker image, based on debian jessie.
 FROM launcher.gcr.io/google/debian8
 
-# https://yarnpkg.com/en/docs/install#linux-tab
-RUN apt-get update && \
-    apt-get install -y curl apt-transport-https git bzip2 build-essential && \
+# Install updates and dependencies
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates libkrb5-dev imagemagick apt-transport-https && \
```

Comment:
What is the `apt-transport-https` for? I don't see it as part of the `npm` builder

Reply:
It was required in order for apt to fetch https dependencies

Reply:
And it was in the original yarn cloud builder since it's required in order to install yarn

Reply:
sg